server/schema/squiggle/squiggle.datasources.js

```diff
@@ -41,11 +41,26 @@ const create = async (squiggleType, content) => {
   }
 };
 
+const updateContent = async (id, newContent) => {
+  try {
+    const _squiggle = await SquiggleModel.findById(id);
+
+    _squiggle.content = newContent || _squiggle.content;
+
+    await _squiggle.save();
+
+    return _squiggle;
+  } catch (error) {
+    throw APIError(null, error);
```

Comment:
@Srish-ty also write a error message, "fail to update squiggle content".

Reply:
resolved. 
 [commit](https://github.com/Monday-Morning/project-reclamation/pull/207/commits/1fade3a73cd4b0c355a269d153505abda9de6745)